plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -621,4 +630,15 @@ public void run() {
       }
     }
   }
+
+  @Override
+  public Image getImage() {
+    return pipelineImage;
+  }
+
+  @Override
+  public void dispose() {
+    pipelineImage = null;
```

Comment:
This means any subsequent uses will not have an tab image.

Reply:
ha ...  Yes, you are right